src/main/js/dynamic/GlobalConfiguration.js

```diff
@@ -6,7 +6,11 @@ import FieldsPanel from 'field/FieldsPanel';
 import ConfigurationLabel from 'component/common/ConfigurationLabel';
 
```

Comment:
The changes here are all linting. I originally thought I had to modify something here, but did not end up needing to. I decided to leave the linting as an incremental improvement.

Reply:
Just saw this message, the linter made some questionable function names that we will probably want to change.

Reply:
we just have to make sure that that function names mapDispatchToProps section aren't the same as the function names used in the dispatch functions in the same section.  That is the linting error called shadowing if my memory is correct.